Assignment 2

hoseinaliyari opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
submitting second assignment

## What did you learn from the changes you have made?
reviewed material on numpy.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
no other approach

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I don't understand the point of having different csv files for this assignment if we are using only the first time,
unless the rest are gonna be used in future assignments

## How were these changes tested?
tested by running the blocks, by checking the third file (all_paths[2]), detect_problems function returns True which by looking at the file, on the last row, a faulty set of data for a patient with all zeros as values can be seen, proving the validity of the function
## A reference to a related issue in your repository (if applicable)

## Checklist
- [x] I can confirm that my changes are working as intended

Commit:
Added my assignmet-1 notebook file to the branch assignment-1

Commit:
Merge branch 'main' of https://github.com/hoseinaliyari/python into assignment-2

Commit:
Submitting my assignemnt

pedram-aliniaye-asli reviewed (changes requested):
You did a great job here!
Just a few quick notes:
- Please add comments to your code to make it clearer for other people and explain what is happening in your code.
- There is a problem in the program because the result of the `detect_problems` function on the first file is True; As you can see in the comments (# Your output for the first file should be False) on the last cell it should be `False'. So go ahead and see what the error is. 
Please go ahead and do the modifications and then let me know. Thanks!

Commit:
edited the assignment based on the feedback

hoseinaliyari commented:
@pedram-aliniaye-asli Hi Pedram, thanks for the feedback.
I added some comments and also checked the code to see why it produces 'True' for the last part. I didn't change anything but by running all blocks again It produced 'False' as expected. so my guess is The 'True' that was seen below it, had to be produced before making the final edition before I submitted it. I mean I probably, before submission, had 'True' as the result, fixed the code but didn't run it to display the corrected output which is 'False'. But I don't know if the outputs of blocks are saved inside the notebooks or not.
